code/modules/clothing/shoes/cowboy.dm

```diff
@@ -6,6 +6,10 @@
 	custom_price = PAYCHECK_CREW
 	var/max_occupants = 4
 	can_be_tied = FALSE
+	/// Do these boots have spur sounds?
+	var/has_spurs = FALSE
+	/// The jingle jangle jingle of our spurs
+	var/list/spur_sound = list('sound/effects/footstep/spurs1.ogg'=1,'sound/effects/footstep/spurs2.ogg'=1,'sound/effects/footstep/spurs3.ogg'=1)
```

Comment:
document this

Reply:
It is documented???

Reply:
it is not in the PR body, you need to announce all the features you added in code (in this case spur jingles)

Reply:
it is, I said you get a cowboy outfit